wordpress-mobile/WordPress-iOS
# Activity Log: add icons to and color them according to the activity status

**Another step for** #7832 

Done on top of https://github.com/wordpress-mobile/WordPress-iOS/pull/8203 and not to be merged until that one is. 

**To test:**

Go to the activity log and check that the little icon displayed matches the web (both the image and the color).

![simulator screen shot - iphone 8 - 2017-11-23 at 15 33 20](https://user-images.githubusercontent.com/5558824/33185835-0aa628b8-d064-11e7-8f10-56af0dcb0ec8.png)
